protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.45.3 - unreleased
+
+- Fix a bug in `Handler::poll` that could result in the handling of events, received
+  by `on_behaviour_event`, to be delayed or completely blocked.
+  See [PR 4961](https://github.com/libp2p/rust-libp2p/pull/4961).
+
 ## 0.45.2
```

Comment:
`0.45.2` hasn't been released yet so you can add the above entry here!

See https://github.com/libp2p/rust-libp2p/pull/4894.